pkgs/development/python-modules/autobahn/default.nix

```diff
@@ -1,34 +1,80 @@
-{ lib, buildPythonPackage, fetchPypi, isPy3k,
-  six, txaio, twisted, zope_interface, cffi, trollius, futures,
-  mock, pytest, cryptography, pynacl
+{ lib
+, argon2_cffi
+, attrs
+, buildPythonPackage
+, cbor
+, cbor2
+, cffi
+, cryptography
+, fetchPypi
+, flatbuffers
+, futures
+, mock
+, msgpack
+, passlib
+, pynacl
+, pytest-asyncio
+, pytestCheckHook
+, pythonOlder
+, six
+, trollius
+, twisted
+, txaio
+, ujson
+, zope_interface
 }:
+
 buildPythonPackage rec {
   pname = "autobahn";
-  version = "20.12.3";
+  version = "21.2.1";
```

Comment:
latest is 21.3.1, maybe update the PR and feel free to cherry-pick https://github.com/NixOS/nixpkgs/pull/116971/commits/e99732a96cf6715277db7d1db89763b37150611c into this PR

Reply:
It's now for 21.3.1 but as a later release of cryptography is required it has to go to `staging`.